src/test/scala/nl.knaw.dans.easy.bag2deposit/ddm/RewriteSpec.scala

```diff
@@ -330,4 +340,34 @@ class RewriteSpec extends AnyFlatSpec with SchemaSupport with Matchers with DdmS
       )))
     // content of the <inCollection> element is validated in CollectionsSpec.collectionDatasetIdToInCollection
   }
+  it should "split archis nrs" in {
+    val ddmIn = ddm(title = "blabla", audience = "D37000", dcmi =
+        <ddm:dcmiMetadata>
+          <dct:identifier scheme="blabla">411047; 411049; 411050 (Archis-vondstmeldingsnr.)</dct:identifier>
+          <dct:identifier>52427; 52429; 52431; 52433; 52435; 52437; 52439; 52441; 52462 (RAAP) (Archis waarneming)</dct:identifier>
```

Comment:
could you add an example with 
```suggestion
          <dct:identifier>52427; 52429; 52431; 52433; 52435; 52437; 52439; 52441; 52462 (RAAP) (Archis waarneming)</dct:identifier>
          <dct:identifier xsi:type="id-type:ARCHIS-WAARNEMING">441832; 1234</dct:identifier>
```
there are more id-types that are ARCHIS numbers, see https://easy.dans.knaw.nl/schemas/vocab/identifier-type.xsd